From 6c56714cc0c2b441da30c0498b31d72fa0068615 Mon Sep 17 00:00:00 2001
From: Kondapally Kalyan <kalyan.kondapally@intel.com>
Date: Sat, 31 Aug 2013 01:47:46 +0300
Subject: [PATCH] Most of our changes in glsurfaceegl are obsolete after [1]
 recent changes to surface factory api. Changes still
 present are in return type of EGLDisplay and EGLConfig.

[1] https://chromiumcodereview.appspot.com/23438002
---
 ui/gl/gl_surface_egl.cc |    5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/ui/gl/gl_surface_egl.cc b/ui/gl/gl_surface_egl.cc
index 5f4edf4..4414d17 100644
--- a/ui/gl/gl_surface_egl.cc
+++ b/ui/gl/gl_surface_egl.cc
@@ -115,8 +115,7 @@ bool GLSurfaceEGL::InitializeOneOff() {
     LOG(ERROR) << "OZONE failed to initialize hardware";
     return false;
   }
-  g_native_display = reinterpret_cast<EGLNativeDisplayType>(
-      surface_factory->GetNativeDisplay());
+  g_native_display = surface_factory->GetNativeDisplay();
 #else
   g_native_display = EGL_DEFAULT_DISPLAY;
 #endif
@@ -140,7 +139,7 @@ bool GLSurfaceEGL::InitializeOneOff() {
     EGL_GREEN_SIZE, 8,
     EGL_RED_SIZE, 8,
     EGL_RENDERABLE_TYPE, EGL_OPENGL_ES2_BIT,
-    EGL_SURFACE_TYPE, EGL_WINDOW_BIT | EGL_PBUFFER_BIT,
+    EGL_SURFACE_TYPE, EGL_WINDOW_BIT,
     EGL_NONE
   };
 
-- 
1.7.9.5

